plugins/dashboard/faucet_routes.go

```diff
@@ -1,77 +0,0 @@
-package dashboard
```

Comment:
@jonastheis do we want to get rid of the faucet request functionality all together from the dashboard? Shouldn't we delete the Faucet page as well then? As I see it is only the dashboard local faucet api that got removed.